server/neptune/gateway/server.go

```diff
@@ -293,11 +293,11 @@ func NewServer(config Config) (*Server, error) {
 		return nil, errors.Wrap(err, "creating github client creator")
 	}
 
-	rootConfigBuilder := &deployCfg.RootConfigBuilder{
+	rootConfigBuilder := &rootConfig.RootConfigBuilder{
```

Comment:
rootConfig.RootConfigBuilder sounds redundant and verbose, can you simplify this?

Reply:
how about `gateway_cfg`?